server/config/routes/api/v21.rb

```diff
@@ -14,6 +14,10 @@
             put :upload
           end
         end
+
+        scope 'unlogged' do
```

Comment:
If we need to use a route path more discreet than "unlogged" like "q" or something, let me know.